### DISCUSSION TITLE
Moving windows from one screen to another (Multi-monitor mouse dragging)

### DISCUSSION THREAD

**Discussion by Winours** [rustdesk/rustdesk#7427] [Q&A] [answered]
Hello everybody,
Rust desk is a really cool project. The only thing I can see to make it perfect would be the multi monitor windows management. I compare with teamviewer which is the only alternative to have the right behavior in this case.

Host has 3 screens (but it is the same with two screens)
Client has 3 screens.

I open the 3 screens of the host on client side and dispatch them on the 3 screens I have locally.
When I try to move a window to another screen, it does not work. Rust desk thinks I want to go to the edge of the screen and host is providing me with windows management as if I had only one screen...
I would be so great to obtain the same behavior as `teamviewer` : you have ponctually acces to windows management, but very quicly it understand you are on another screen and allow you to move a window from one screen to another.

Hope this is clear as english is not my native language.

Best regards

**Comment by coelho** [comment]
Seeing this behavior as well on Nightly. Dragging windows between screens doesn't behave properly.
- 2 Windows 11 Machines (Host & Client)
- Both Dual Monitors
- All Monitors at 1920x1080 Resolution

**Comment by rustdesk** [comment]
A good example of how to get your feature request higher priority.
 https://twitter.com/rustdesk/status/1770643072118095914

**Reply by coelho** [reply]
@rustdesk 
1. This is a bug report, not a feature request.
2. We are a paying customer and supporter of RustDesk.
3. We ask that the issue remain open. If it becomes high priority, we will fund it, but currently we submitted for your team.

**Reply by coelho** [reply]
I appreciate you @rustdesk ! thank you

**Reply by Winours** [reply]
On my version (nightly) I cannot see any difference. Is it normal ?

**Comment by rustdesk** [comment, marked as answer]
Fixed in https://github.com/rustdesk/rustdesk/pull/7844

and ready now https://twitter.com/rustdesk/status/1784102193760801170
